pkgs/development/libraries/geoclue/default.nix

```diff
@@ -30,27 +33,21 @@ stdenv.mkDerivation rec {
 
   propagatedBuildInputs = [ glib glib-networking ];
 
-  # Whitelist elementary's agent
-  patches = [
-    (fetchpatch {
-      url = "https://gitlab.freedesktop.org/geoclue/geoclue/commit/2b0491e408be1ebcdbe8751bb2637c1acb78f71e.patch";
-      sha256 = "0pac94y55iksk340dlx3gkhb9lrci90mxqqy5fnh1zbjw9bqxfn4";
-    })
-  ];
-
-  configureFlags = [
-    "--with-systemdsystemunitdir=$(out)/etc/systemd/system"
-    "--enable-introspection"
-    "--enable-gtk-doc"
-    "--enable-demo-agent=${if withDemoAgent then "yes" else "no"}"
+  mesonFlags = [
+    "-Dsystemd-system-unit-dir=${placeholder "out"}/etc/systemd/system"
+    "-Ddemo-agent=${if withDemoAgent then "true" else "false"}"
```

Comment:
This is default true now IIRC. It should be fine if I drop this completely(and the option)?
I wasn't sure because we were defaulting to false for some reason.

Reply:
We do not want to have it in GNOME or Elementary, I think.

Reply:
Going with that.